### ISSUE TITLE
Finetuning Code and Data Release

### ISSUE THREAD

**Issue by XinrunXu** [THUDM/Android-Lab#1]
Great work on ANDROIDLAB! I'm interested in reproducing the finetuning results.  

Could you please share the code and the subset of the Android Instruction dataset (726 traces, 6208 steps) used for finetuning on the benchmark tasks?  

This would be very helpful for the community.

**Comment by xuyifan-0731** [comment]
Thank you very much for your interest in our work!

Regarding Finetuning Code:
For LLMs (qwen2, chatglm4, and llama3.1) and qwen2-vl, we used full-scale finetuning via [llama-factory](https://github.com/hiyouga/LLaMA-Factory) for training.
For cogvlm and llama3.2-vision, we conducted full-scale finetuning using [ms-swift](https://github.com/modelscope/ms-swift).

All training was performed with a learning rate of 1e-5, training for 3 epochs, and a batch size of 128. 

Regarding Data Release:
We are currently undergoing an ethics review for the dataset and plan to release the Android Instruction dataset (726 traces, 6208 steps) next week.

**Comment by cobraheleah** [comment]
> Thank you very much for your interest in our work!
> 
> Regarding Finetuning Code: For LLMs (qwen2, chatglm4, and llama3.1) and qwen2-vl, we used full-scale finetuning via [llama-factory](https://github.com/hiyouga/LLaMA-Factory) for training. For cogvlm and llama3.2-vision, we conducted full-scale finetuning using [ms-swift](https://github.com/modelscope/ms-swift).
> 
> All training was performed with a learning rate of 1e-5, training for 3 epochs, and a batch size of 128.
> 
> Regarding Data Release: We are currently undergoing an ethics review for the dataset and plan to release the Android Instruction dataset (726 traces, 6208 steps) next week.

Hello，when will the training set be released? We would like to reproduce the results.

**Comment by enduringstack** [comment]
Hello，the question, when will the training set be released? We would like to reproduce the results.

**Comment by xuyifan-0731** [comment]
We have also open-sourced the Android Instruct dataset mentioned in the paper. Please refer to docs/instruction_tuning.md for more details.

**Comment by XinrunXu** [comment]
Hello，the question, when will the finetuning code set be released? We would like to reproduce the results.
